src/configs/base_report_templates.c4m

```diff
@@ -1907,6 +1912,26 @@ report_template usage {
   ~key._OP_PLATFORM.use                       = true
 }
 
+report_template heartbeat {
+  shortdoc: "Small report for periodic heartbeats"
+  doc: """
+Very small report template for sending heartbeats.
+Normally exec report should collect most of the information
+about the chalk runtime. After that point heartbeat report
+only indicates liveliness of the process so it can be very
+minimal.
+"""
+  ~key.METADATA_ID.use                        = true
+  ~key.CHALK_ID.use                           = true
+  ~key._CHALKS.use                            = true
+  ~key._OPERATION.use                         = true
+  ~key._TIMESTAMP.use                         = true
+  ~key._DATETIME.use                          = true
+  ~key._ACTION_ID.use                         = true
+  ~key._EXEC_ID.use                           = true
+  ~key._OP_ARGV.use                           = true
```

Comment:
not sure if this key is strictly necessary although seems like a useful sanity check when viewing the report that its for the correct execution? as otherwise `_EXEC_ID` is a random string so not super friendly...

thoughts whether we should keep it in or remove it from heartbeat reports?